### PR TITLE
IBX-615: As an Editor, I want to see redesigned checkbox and radio buttons in all states

### PR DESCRIPTION
**Pull request by GrabowskiM**
JIRA ref: https://issues.ibexa.co/browse/IBX-615

https://github.com/ezsystems/date-based-publisher/pull/227
https://github.com/ezsystems/ezcommerce-admin-ui/pull/92
https://github.com/ezsystems/ezcommerce-shop/pull/323
https://github.com/ezsystems/ezplatform-admin-ui/pull/1793
https://github.com/ezsystems/ezplatform-calendar/pull/76
https://github.com/ezsystems/ezplatform-form-builder/pull/286
https://github.com/ezsystems/ezplatform-matrix-fieldtype/pull/55
https://github.com/ezsystems/ezplatform-page-builder/pull/777
https://github.com/ezsystems/ezplatform-page-fieldtype/pull/189
https://github.com/ezsystems/ezplatform-personalization/pull/212
https://github.com/ezsystems/ezplatform-segmentation/pull/30
https://github.com/ezsystems/ezplatform-site-factory/pull/126

### CODE REVIEW

File: src/bundle/Resources/public/scss/_inputs.scss [hunk 1]
```diff
@@ -5,32 +5,42 @@
...
+    .ibexa-input {
+        position: relative;
     }
 }
 
-.ez-input {
+.ibexa-input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_inputs.scss:16]: not sure if this should be on all inputs or just checkbox/radio

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_inputs.scss [hunk 2]
```diff
@@ -5,32 +5,42 @@
...
+    position: relative;
+    display: inline-block;
     outline: none;
     transition: all 0.2s $ibexa-admin-transition;
```
Comment (lucasOsti) [inline src/bundle/Resources/public/scss/_inputs.scss:20]: `$ibexa-admin-transition-duration`

File: src/bundle/Resources/public/scss/_inputs.scss [hunk 3]
```diff
@@ -69,6 +98,33 @@
...
+            box-shadow: inset 0 0 0 calculateRem(4px) $ibexa-color-primary;
+
+            &:disabled {
+                border-color: $ibexa-color-primary-300;
+                box-shadow: inset 0 0 0 calculateRem(4px) $ibexa-color-primary-300;
+            }
+
+            &:active,
+            &:focus {
+                &:after {
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_inputs.scss:115]:
```suggestion
                &::after {
```
ref: https://developer.mozilla.org/en-US/docs/Web/CSS/::after#syntax

**Review by tischsoic — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_inputs.scss [hunk 4]
```diff
@@ -5,32 +5,46 @@
...
+    background-color: $ibexa-white;
+
+    &:hover {
+        border-color: $ibexa-color-primary;
+        cursor: pointer;
+    }
+
+    &:active,
+    &:focus {
+        border-color: $ibexa-color-primary;
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_inputs.scss:28]: Not all active inputs have this border colour. Actually, not sure if any? Checkboxes do not have `:active` state, do they?

File: src/bundle/Resources/public/scss/_inputs.scss [hunk 5]
```diff
@@ -5,32 +5,46 @@
...
+        box-shadow: 0 0 calculateRem(3px) 0 rgba($ibexa-color-primary, 0.75);
+    }
 
     &:disabled {
-        opacity: 0.3;
+        cursor: default;
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_inputs.scss:33]:
```suggestion
        cursor: not-allowed;
```

**Review by lucasOsti — APPROVED** [review]

**Review by tischsoic — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Pull request merged**